src/app/OperationalDeviceProxy.cpp

```diff
@@ -121,7 +120,6 @@ CHIP_ERROR OperationalDeviceProxy::UpdateDeviceData(const Transport::PeerAddress
         if (secureSession != nullptr)
         {
             secureSession->SetPeerAddress(addr);
-            secureSession->SetMRPIntervals(mrpIdleInterval, mrpActiveInterval);
```

Comment:
No I asked Pankaj, we shouldn't update session MRP parameter.

Because for secure sessions, MRP parameters are negotiated from CASE session PBKDF parameters, and the new parameters are come from mDNS discover, which is less trust-able than CASE pairing.